examples/complete/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM alpine
```

Comment:
I didn't see any reference to this example in the PR, so I'd remove it because it has baked in seed version 0.0.3 and no algorithm implementation. I'd like to keep examples in this repo functional, e.g. `extractor` / `addition-algorithm`.